services/QuillLMS/client/app/bundles/Teacher/components/subscriptions/current_subscription.jsx

```diff
@@ -19,18 +20,15 @@ export default class CurrentSubscription extends React.Component {
 
   onceYourPlanExpires() {
     const { subscriptionType, } = this.props
-    return `Once your current ${subscriptionType} Premium subscription expires, you will be downgraded to the Quill Basic subscription.`;
+    return `Once your current ${subscriptionType} subscription expires, you will be downgraded to the Quill Basic subscription.`;
   }
 
   getCondition() {
     const { subscriptionType, } = this.props
-    switch (subscriptionType) {
-      case 'School':
-        return 'school';
-      case 'School Sponsored':
-        return 'school sponsored';
-      default:
-        return 'other';
+    if (['School Premium', 'District Premium'].includes(subscriptionType)) {
```

Comment:
We should make these their own constants in that `constants.js` file and then import them here since they're used for logic.